_im-plugin/ism/api.md

```diff
@@ -546,3 +546,154 @@ DELETE _plugins/_ism/policies/policy_1
   "_primary_term": 1
 }
 ```
+
+## Error prevention validation
+Introduced 2.4
+{: .label .label-purple }
+
+ISM allows you to execute an action automatically. However, action execution can fail for a variety of reasons. Error prevention validation checks the possible reason for a failure before execution of an action.
```

Comment:
However, "an action can fail"? You can use error prevention validation to dry run your request to rule out failures?

Reply:
I see now. It's not to dry run a request. So, it's verifying the actions that are already set up by ISM? Then it should be "to dry run an action", not request.